esp-hal/src/gpio/mod.rs

```diff
@@ -31,6 +31,9 @@
 //! GPIO interrupts. For more information, see the
 //! [`Io::set_interrupt_handler`].
 //!
+//! This driver also implements pin-related traits from [embedded-hal] and
+//! [Wait] trait from [embedded-hal-async].
```

Comment:
When we're talking about a crate we already depend on, it's better to use the rustdoc syntax for linking. In this case you can write `[Wait](embedded_hal_async::digital::Wait)` and rustdoc will generate the correct link to the trait for you.

More info on the syntax here: https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html

Reply:
> and rustdoc will generate the correct link to the trait for you.

Alas, we don't generate docs for dependencies and we're hosting them ourselves, so this doesn't actually work 🫠

Reply:
If we don't build docs for deps, it will link to docs.rs iirc, at least it did in this case when I tried it locally :sweat_smile: